go/chat/attachments/s3.go

```diff
@@ -351,27 +307,6 @@ func (a *S3Store) uploadPart(ctx context.Context, task *UploadTask, b job, previ
 	return nil
 }
 
-// putRetry sends a block to S3, retrying retryAttempts times w/ backoff.
```

Comment:
I feel like this one was useful during multipart uploads to just get one part uploaded during a transient failure.  Have you tested multipart upload retries without this?

Reply:
I think if there is a transient failure, the goamz library will pick it up anyway. But even if it doesn't, we will be retrying this upload again from the `Deliverer` retry, which will kick in right away. It's just this way it counts as a "failure" over in the `Deliverer` world.

Reply:
ok